CMake/external_pybind11.cmake

```diff
@@ -84,3 +81,19 @@ endfunction()
 
 # Trigger the build
 get_pybind11()
+
+# This function override "pybind11_add_module" function,  arguments is same as "pybind11_add_module" arguments
+# pybind11_add_module(<name> SHARED [file, file2, ...] )
+# Must be declared after pybind11 configuration above
+# "_pybind11_add_module" is calling the origin pybind11 function
```

Comment:
I'd place this on the actual line instead of here

Reply:
Accepted